unitypack/export.py

```diff
@@ -147,7 +147,7 @@ def __init__(self, mesh):
 	@staticmethod
 	def face_str(indices, coords, normals):
 		ret = ["f "]
-		for i in indices[::-1]:
```

Comment:
I believe this was done to convert from left-handed coordinates to right-handed coordinates, FWIW.

Reply:
Oh, um, TBH I thought pull requests would only add the commits from before they were made. 
Either way, you are correct, however removing the `-` on line 31 inverts the models once again, so the face normals are as they should.